lxd/networks_utils.go

```diff
@@ -1548,3 +1548,17 @@ func networkDeviceBind(pciSlotName string, driverPath string) error {
 
 	return nil
 }
+
+// networkDeviceBindWait waits for network interface to appear after being binded.
+func networkDeviceBindWait(devName string) error {
+	for i := 0; i < 10; i++ {
+		if shared.PathExists(fmt.Sprintf("/sys/class/net/%s", devName)) {
+			return nil
+		}
+
+		logger.Infof("Waiting for interface \"%s\" to be bound...", devName)
```

Comment:
This looks like debug logging at best and even at debug level this is likely to spam, given we just wait 500ms total, I'd just wait for the error